eng/build.ps1

```diff
@@ -306,7 +310,8 @@ function GetIbcDropName() {
     $branch = GetIbcSourceBranchName
     Write-Host "Optimization data branch name is '$branch'."
 
-    $drop = Find-OptimizationInputsStoreForBranch -ProjectName "DevDiv" -RepositoryName "VS" -BranchName $branch
+    $pat = ConvertTo-SecureString $officialDevDivDropAccessToken -AsPlainText -Force
```

Comment:
>ConvertTo [](start = 11, length = 9)

This is suspicious: [https://github.com/dotnet/platform-compat/blob/master/docs/DE0001.md](https://github.com/dotnet/platform-compat/blob/master/docs/DE0001.md)

Reply:
`Find-OptimizationInputsStoreForBranch` only takes SecureString for PAT. That said, it'd be great if someone can confirm the way I handle the token here is OK
https://devdiv.visualstudio.com/Engineering/_git/Validation?path=%2Fsrc%2FOptimization%2FData.PowerShell%2FAuthenticatedCmdlet.cs&version=GBmain&line=51&lineEnd=52&lineStartColumn=1&lineEndColumn=1&lineStyle=plain

Reply:
@GrabYourPitchforks Should this be addressed in `Find-OptimizationInputsStoreForBranch`?